src/ucp/core/ucp_context.c

```diff
@@ -376,6 +376,10 @@ static ucs_config_field_t ucp_context_config_table[] = {
    "another thread, or incoming active messages, but consumes more resources.",
    ucs_offsetof(ucp_context_config_t, flush_worker_eps), UCS_CONFIG_TYPE_BOOL},
 
+  {"SHUFFLE_WORKER_EPS", "n",
+   "Enable shuffling endpoints while creating a workers",
```

Comment:
while creating EPs actually

Reply:
Maybe `Randomize order of newly created endpoints in the internal list` instead.
@yosefe wdyt?

Reply:
done